C/Mergesort.c

```diff
@@ -0,0 +1,77 @@
+
+
+
+
+#include<stdio.h>
+#include<stdlib.h>
+
+void merge(int arr[], int l, int m, int r) 
+{ 
+    int i, j, k; 
+    int n1 = m - l + 1; 
+    int n2 =  r - m;    
+    int L[n1], R[n2];   
+    for (i = 0; i < n1; i++) 
+        L[i] = arr[l + i]; 
+    for (j = 0; j < n2; j++) 
+        R[j] = arr[m + 1+ j];   
+    i = 0; 
+    j = 0; 
+    k = l; 
+    while (i < n1 && j < n2) 
+    { 
+        if (L[i] <= R[j]) 
+        { 
+            arr[k] = L[i]; 
+            i++; 
+        } 
+        else
+        { 
+            arr[k] = R[j]; 
+            j++; 
+        } 
+        k++; 
+    } 
+      while (i < n1) 
+    { 
+        arr[k] = L[i]; 
+        i++; 
+        k++; 
+    } 
+      while (j < n2) 
+    { 
+        arr[k] = R[j]; 
+        j++; 
+        k++; 
+    } 
+} 
+  void mergeSort(int arr[], int l, int r) 
+{ 
+    if (l < r) 
+    { 
+        
+        int m = l+(r-l)/2;  
+       
+        mergeSort(arr, l, m); 
+        mergeSort(arr, m+1, r); 
+  
+        merge(arr, l, m, r); 
+    } 
+}
+int main(){
+    int *arr,n,i;
+
+    printf("Enter the size of the array :");
+    scanf("%d",&n);
+    printf("Enter the array elements :\n ");
+    for(i=0;i<n;i++){
+        scanf("%d",&arr[i]);
+    }
+    mergeSort(arr,0,n-1);
+    printf("Sorted Array :\t");
+    for(i=0;i<n;i++){
+        printf("%d ",arr[i]);
+    }
+
+
+}
```

Comment:
Add Sample I/P, O/P, Time and Space complexity in a single MULTILINE COMMENT.
Let me know, once done

Reply:
I have done that can I create new PR now?

Reply:
Yes, create a new PR from a new branch.
Closing this for now.

Reply:
I have created Pull request